F1/F1/Race/Repository/RaceRepository.swift

```diff
@@ -19,7 +19,19 @@ protocol RaceRepositoryType: AnyObject {
 class RaceRepository: RaceRepositoryType {
 
     func fetchRaceResults(completion: @escaping RaceResults) {
-        let url = Endpoints.racing
-        URLSession.shared.request(endpoint: url, method: .GET, completion: completion)
+        if let savedRaces = CoreDataManager.shared.fetchRaces(), !savedRaces.isEmpty {
```

Comment:
what if there is new data to get? 
if you wnt to cater for offline better, look int something called Reachability